dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueStatus.java

```diff
@@ -30,24 +30,34 @@
 /**
  * Enum for description of repeatable stage value
  */
-public enum RepeatableStageValueStatus
+public enum ValueStatus
 {
     //repeating of the repeatable stage does not exist
-    UNDEFINED,
+    NOT_DEFINED,
 
     //value is not set
-    UNSET,
+    NOT_SET,
 
     //value is set
     SET;
 
-    public static RepeatableStageValueStatus of( boolean isDefined, boolean isSet )
+    public static ValueStatus of( boolean isDefined, boolean isSet )
     {
         if ( !isDefined )
         {
-            return RepeatableStageValueStatus.UNDEFINED;
+            return ValueStatus.NOT_DEFINED;
         }
 
-        return isSet ? RepeatableStageValueStatus.SET : RepeatableStageValueStatus.UNSET;
+        return isSet ? ValueStatus.SET : ValueStatus.NOT_SET;
+    }
+
+    public static String toString( ValueStatus valueStatus )
```

Comment:
Instead of associating the "abbreviations" here, would probably be better to associate them with the enums themselves. Like it's done in this enum, for example:
```
DataDimensionType.java
```

Reply:
Done